src/commands/react-native/README.md

```diff
@@ -93,9 +102,79 @@ This allows Datadog to create proper URLs such as:
 | GitHub or GitLab  | https://\<repository-url\>/blob/\<commit-hash\>/\<tracked-file-path\>#L\<line\> |
 | Bitbucket | https://\<repository-url\>/src/\<commit-hash\>/\<tracked-file-path\>#lines-\<line\>  |
 
+
+### `xcode`
+
+This command can be called from an xcode build phase to execute the react-native bundle command and then upload the sourcemaps.
+The upload only happens when your target has a "Release" build configuration to prevent overwriting existing sourcemaps.
```

Comment:
```suggestion

The upload only happens when your target has a "Release" build configuration that prevents overwriting existing source maps.
```

Reply:
I think people could understand it as:

```
The upload only happens when your target has a "Release" build configuration that contains something that prevents overwriting existing source maps.
```
Whereas what I meant is:
```
The upload only happens when your target has a "Release" build configuration; that prevents overwriting existing source maps when running a build with another configuration such as "Debug".
```
Are you ok with my new formulation?